server/task_executor_pitr_cutover.go

```diff
@@ -93,27 +94,44 @@ func (*PITRCutoverTaskExecutor) pitrCutover(ctx context.Context, task *api.Task,
 	}
 	defer driver.Close(ctx)
 
-	driverDB, err := driver.GetDBConnection(ctx, "")
-	if err != nil {
-		return true, nil, err
-	}
-	conn, err := driverDB.Conn(ctx)
-	if err != nil {
-		return true, nil, err
-	}
-	defer conn.Close()
-
-	log.Debug("Swapping the original and PITR database", zap.String("originalDatabase", task.Database.Name))
-	pitrDatabaseName, pitrOldDatabaseName, err := mysql.SwapPITRDatabase(ctx, conn, task.Database.Name, issue.CreatedTs)
-	if err != nil {
-		log.Error("Failed to swap the original and PITR database", zap.String("originalDatabase", task.Database.Name), zap.String("pitrDatabase", pitrDatabaseName), zap.Error(err))
-		return true, nil, fmt.Errorf("failed to swap the original and PITR database, error: %w", err)
-	}
-	log.Debug("Finished swapping the original and PITR database", zap.String("originalDatabase", task.Database.Name), zap.String("pitrDatabase", pitrDatabaseName), zap.String("oldDatabase", pitrOldDatabaseName))
-
-	backupName := fmt.Sprintf("%s-%s-pitr-%d", api.ProjectShortSlug(task.Database.Project), api.EnvSlug(task.Database.Instance.Environment), issue.CreatedTs)
-	if _, err := server.scheduleBackupTask(ctx, task.Database, backupName, api.BackupTypePITR, api.SystemBotID); err != nil {
-		return true, nil, fmt.Errorf("failed to schedule backup task for database %q after PITR, error: %w", task.Database.Name, err)
+	switch task.Instance.Engine {
```

Comment:
Why not doing similar to what we have in restore.go that does pitrCutover() and postgresPitrCutover()?